.github/workflows/ci.yml

```diff
@@ -45,6 +39,13 @@ jobs:
         ./configure
         make
 
+    - name: Install dependencies
+      run: |
+        php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
+        php composer-setup.php
```

Comment:
To have composer.phar in the same directory. So, that we don't need to script to get the path of composer.phar.

Reply:
installing composer twice seems like it could have weird consequences. let's just find the path. maybe something like?:
```
# get path to composer.phar so we can run it with custom php.ini
COMPOSER_PHAR=$(realpath $(which composer))
php -c php.ini $COMPOSER_PHAR require ...
```